payment_alipay/__openerp__.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+
+{
+    'name': 'Alipay Payment Acquirer',
+    'category': 'Website',
+    'summary': 'Payment Acquirer: Alipay Implementation',
+    'version': '1.0',
```

Comment:
For version numbers please check:
* Contribution guidelines: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md